src/lib.rs

```diff
@@ -403,11 +386,6 @@ impl Check {
 
     pub fn check_release(&self) -> anyhow::Result<Report> {
         let mut config = GlobalConfig::new().set_level(self.log_level);
-        // we don't want to set auto if the choice is not set because it would overwrite
-        // the value if the CARGO_TERM_COLOR env var read in GlobalConfig::new() if it is set
-        if let Some(choice) = self.color_choice {
-            config = config.set_color_choice(choice);
-        }
```

Comment:
It's a bit of a big (and semver-breaking) change, but would you be opposed to taking a `GlobalConfig` (or a `&mut GlobalConfig`) as a parameter to the `check_release` function?  Otherwise, it's possible, but we would need to double our configuration API - have one call for `Check` and one call for `GlobalConfig`.

Reply:
Sure, we can do that. Do you think taking it by value or by `&mut` is better?

Reply:
I made it take `&mut` to leave the option open for a library user to run multiple `Check`s